test the local compiled version, not the system installed version

http://bugs.gentoo.org/215359

--- util/defs.in
+++ util/defs.in
@@ -67,6 +67,7 @@
 export SHELL
 test -z "$TEXI2DVI" && TEXI2DVI=`cd $srcdir && pwd`/texi2dvi
 
+PATH=$PWD:$PATH
 echo $PATH
 # Some shells forget to export modified environment variables.
 # (See note about `export' in the Autoconf manual.)
